technical-reports/format/types.md

```diff
@@ -58,7 +58,7 @@ $translucent-shadow: hsla(300, 100%, 50%, 0.5);
 
 ## Dimension
 
-Represents an amount of distance in a single dimension in the UI, such as a position, width, height, radius, or thickness. The `$type` property MUST be set to the string `dimension`. The value must be a string containing a number (either integer or floating-point) followed by either a "px" or "rem" unit (future spec iterations may add support for additional units).
+Represents an amount of distance in a single dimension in the UI, such as a position, width, height, radius, or thickness. The `$type` property MUST be set to the string `dimension`. The value must be a string containing a number (either integer or floating-point) followed by either a "px" or "rem" unit (future spec iterations may add support for additional units). This includes `0` which also MUST be followed by either a "px" or "rem" unit.
```

Comment:
This is helpful context and may be worth including in the following code block/example. Thoughts?

```jsonc
{
  "spacing-0": {
    "$value": "0rem",
    "$type": "dimension"
  },
  "spacing-1": {
    "$value": "0.25rem",
    "$type": "dimension"
  },
  // etc.
}
```

Reply:
You mean to add the `0rem` to the existing example? I am fine either way, don't know if it is needed, but I think it for sure does not hurt.

Reply:
Yes, updating the existing example 👍 Every "MUST" condition thus far is demonstrated and think this was a great callout that might otherwise be overlooked. Like you mentioned it doesn't hurt to add so I'll trust your judgement on whether or not to include!